apps/ltn/src/app.rs

```diff
@@ -283,22 +285,13 @@ impl App {
         &self.per_map.proposals.current_proposal.partitioning
     }
 
-    pub fn calculate_draw_major_road_labels(&mut self, ctx: &mut EventCtx) {
-        if self.per_map.draw_major_road_labels.is_none() {
-            self.per_map.draw_major_road_labels = Some(DrawSimpleRoadLabels::only_major_roads(
+    pub fn calculate_draw_all_local_road_labels(&mut self, ctx: &mut EventCtx) {
+        if self.per_map.draw_all_local_road_labels.is_none() {
+            self.per_map.draw_all_local_road_labels = Some(DrawSimpleRoadLabels::new(
                 ctx,
                 self,
-                render::colors::ROAD_LABEL,
-            ));
-        }
-    }
-
-    pub fn calculate_draw_all_road_labels(&mut self, ctx: &mut EventCtx) {
-        if self.per_map.draw_all_road_labels.is_none() {
-            self.per_map.draw_all_road_labels = Some(DrawSimpleRoadLabels::all_roads(
-                ctx,
-                self,
-                render::colors::ROAD_LABEL,
+                render::colors::LOCAL_ROAD_LABEL,
+                Box::new(|r| r.get_rank() == osm::RoadRank::Local && !r.is_light_rail()),
```

Comment:
Main/local roads use different label colors, so we want these to be disjoint now